### ISSUE TITLE
Data format for continuous pre-training

### ISSUE THREAD

**Issue by tiendung** [QwenLM/Qwen#394]
thanks for providing SFT scripts, but I found that the data is going to be formatted as instruction / chat finetune format. Details here https://github.com/QwenLM/Qwen/blob/main/finetune.py#L123

I would like to continuous pre-train the model for around 40b tokens before doing final instruction finetune, and would like to ask which part of data format code should I modified to make it possible?

Thanks.

**Comment by JustinLin610** [comment]
Our pretraining follows the implementation of conventional gpt training, which accepts inputs with an eod token as the indicator of the end of document. You can consider concatenating different documents, but our finetuning script does not support this.

**Comment by tiendung** [comment]
> Our pretraining follows the implementation of conventional gpt training, which accepts inputs with an eod token as the indicator of the end of document. You can consider concatenating different documents, but our finetuning script does not support this.

One more question:
After adding `eod` token, do we need to mask it out (attention mask's value = 0) to do continual pre-training?
